### ISSUE TITLE
Cooperation with similar Project

### ISSUE THREAD

**Issue by MatthiasLiszt** [paulfitz/cosmicos#39]
Today I opened a similar project which tries to communicate to ET by using esoteric languages like Thue and Brainfuck. Brainfuck is explained through Thue. If you are interested in cooperating or just have some advice please take a look at the README.md at github.com/matthiasliszt/cosmicDew . 

Thanks in advance!

**Comment by paulfitz** [comment]
Neat! Thue looks interesting, I hadn't heard of it. The README mentions a bf**k directory that I don't see committed?

**Comment by MatthiasLiszt** [comment]
That's right. There is no code in brainfuck yet as I am trying the get for the Thue to Brainfuck Interpreter. I know people wrote C to brainfuck compiler etc. years ago and I really need that stuff if I want that project to progress fast.  Thanks for the fast reply!
